intel_extension_for_transformers/llm/quantization/nn/modules.py

```diff
@@ -130,8 +130,10 @@ def forward(self, x: torch.Tensor):
         return out
 
     def set_weights_bias(self, weight_data, bias=None):
+        shape = weight_data.shape
         weight = torch.ops.jblasop.woq_quantize(
             weight_data, True, self.blocksize, self.compute_dtype, self.weight_dtype, self.scale_dtype)
+        weight.resize_(shape)
```

Comment:
this will lead weight hold shape*sizeof(weight.dtype) bytes which significant waste memory.

Reply:
It's a known issue. @PenghuiCheng  will try to fix it later.